pkg/cloud/services/networking/port.go

```diff
@@ -324,7 +325,29 @@ func getPortName(baseName string, portSpec *infrav1.PortOpts, netIndex int) stri
 	return fmt.Sprintf("%s-%d", baseName, netIndex)
 }
 
-func (s *Service) CreatePorts(eventObject runtime.Object, desiredPorts []infrav1.ResolvedPortSpec, resources *infrav1.MachineResources) error {
+func (s *Service) CreateMachinePorts(eventObject runtime.Object, desiredPorts []infrav1.ResolvedPortSpec, resources *infrav1.MachineResources) error {
```

Comment:
If we make this function take and return []infrav1.ResolvedPortSpec, I'm guessing we won't need to copy it?

Reply:
I need to remove it, it's a leftover.